cli/wallet/wallet.go

```diff
@@ -295,6 +389,23 @@ func createWallet(ctx *cli.Context) error {
 	return nil
 }
 
+func signTx(tx *transaction.Transaction, acc *wallet.Account) {
+	priv := acc.PrivateKey()
+	sign := priv.Sign(tx.GetSignable())
```

Comment:
`GetSignable` doesn't exist yet as of 70a8b764bb917c9898db8e7581f493eadf6ff119, you won't be able to build this commit.